README.md

```diff
@@ -1,4 +1,224 @@
-# cockroach-operator
-Kubernetes Operator for CockroachDB
+# CockroachDB Kubernetes Operator
 
-This project is not production ready and is in an alpha state.
+The CockroachDB Kubernetes Operator deploys CockroachDB on a Kubernetes cluster. You can use the Operator to manage the configuration of a running CockroachDB cluster, including:
+
+- Authenticating certificates
+- Configuring resource requests and limits
+- Scaling the cluster
+- Performing a rolling upgrade
+
+## Limitations
+
+- This project is in alpha and is not production-ready.
+- The Operator currently runs on GKE.
+- Currently the Kubernetes CA is required to deploy a secure cluster.
+- Expansion of persistent volumes is not yet functional.
+
+## Prerequisites
```

Comment:
Here is the compatibility matrix for the operator https://github.com/kubernetes/client-go#compatibility-matrix. We are using 0.18.5 go-client